src/cljs/main/broadfcui/main.cljs

```diff
@@ -15,9 +15,20 @@
    [broadfcui.page.profile :as profile-page]
    [broadfcui.page.status :as status-page]
    [broadfcui.page.workspaces-list :as workspaces]
+   [broadfcui.tooltip :as tooltip]
    [broadfcui.utils :as utils]
    ))
 
+(react/defc TooltipTest
+  {:render
+   (fn []
+     [:div {:style {:marginBottom "2rem"}}
+      "You may need "
+      (tooltip/with-default
+        [:span {:style {:fontStyle "italic"}}
```

Comment:
I'd also want a default "has-tooltip" style for text on which the tip was active.